config/bundle.yaml

```diff
@@ -30408,55 +30408,6 @@ status:
   conditions: []
   storedVersions: []
 ---
-apiVersion: admissionregistration.k8s.io/v1
```

Comment:
We should also change Makefile from

`manifests: controller-gen`

to

`manifests: verify controller-gen`

This way everytime when you run make manifests, it will first format the code in a consistent way to avoid unnecessary changes everytime